lib/clibuddy/runner.rb

```diff
@@ -159,7 +159,11 @@ def lookup_command(name, provided_args)
         # A: we can do differently-formatted exception handling that makes it clear.
         raise Errors::NoSuchCommand.new(name)
       end
-      InterpretedCommand.new(cmd, provided_args)
+      begin
+        InterpretedCommand.new(cmd, provided_args)
+      rescue OptionParser::ParseError => e
+        raise Errors::OptionParserError.new(cmd.name)
```

Comment:
Is there any value in the original message from ParseError?

Reply:
Yes, and we get that with the `e.cause` logic I added into the error printer